src/pages/editor/EditorCurators.tsx

```diff
@@ -37,47 +39,63 @@ export const EditorCurators = () => {
     meta: { errorMessage: t('feedback.error.get_users_for_institution') },
   });
   const curators = curatorsQuery.data ?? [];
+  const filteredCurators = curators.filter((curator) => {
+    const name = `${curator.givenName} ${curator.familyName}`.toLowerCase();
+    return name.includes(searchQuery.toLowerCase());
+  });
 
   // Ensure selected page is not out of bounds due to manipulated userList
-  const validPage = curators.length <= (page - 1) * rowsPerPage ? 1 : page;
-  const curatorsOnPage = curators.slice((validPage - 1) * rowsPerPage, validPage * rowsPerPage);
+  const validPage = filteredCurators.length <= (page - 1) * rowsPerPage ? 1 : page;
+  const curatorsOnPage = filteredCurators.slice((validPage - 1) * rowsPerPage, validPage * rowsPerPage);
 
-  return curatorsQuery.isLoading ? (
```

Comment:
lurer på om jeg har vært i overkant purist tidligere ved å skrive sånn som dette i stedet for å alltid ha en tag (`<>`) ytterst. Blir en veldig mye enklere diff for fremtidige endringer som nå om man alltid har en tag ytterst. Tror jeg er villig til å betale prisen med et ektra innrykk fremover 🤔

Reply:
Hvorfor? I denne PRen har jeg bare lagt til en `<TextField>`, men ser man på PR er det veldig mye annet man må sette seg inn i